vnext/Microsoft.ReactNative/Utils/ManagedResource.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+#include <utility>
+
+template <typename T, typename closeFn_t>
```

Comment:
>closeFn_t [](start = 31, length = 9)

It should be TCloseFunction per Microsoft naming guidelines: [https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces)
DO prefix descriptive type parameter names with T.